### PR TITLE
Baibhav tiwari patch 3

### PR DESCRIPTION
**Pull request by baibhav-tiwari**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3718"></a>

### CODE REVIEW

**Comment by baibhav-tiwari** [comment]
@Aadarsh805 Can you please tell what i have done wrong here

**Review by Tejender1521 — DISMISSED** [review]
Everything looks good and ready to be merged.

**Comment by Aadarsh805** [comment]
> Everything looks good and ready to be merged.

JSON check failed though

File: data/baibhav-tiwari.json [hunk 1]
```diff
@@ -20,5 +21,44 @@
...
+      "icon": "FaGlobe",
+      "color": "black",
+      "description": "i was always interested in creating websites, so i decided to learn web dev."
+    },
+    {
+      "title": "Social Media",
+      "date": "2022",
+      "icon": "FaAndroid",
+      "color": "black",
+      "description": "Firstly i thought Twitter is boring but as i started using it more i started loving it.",
```
Comment (Pradumnasaraf) [inline data/baibhav-tiwari.json:45]:
```suggestion
      "description": "Firstly i thought Twitter is boring but as i started using it more i started loving it."
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, 

Please remove the extra `,`

File: data/baibhav-tiwari.json (hunk 1 above)
Comment (baibhav-tiwari) [reply data/baibhav-tiwari.json]: done , thanks

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**